staging/src/k8s.io/kms/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package kms contains the proto definitions for the kms API.
```

Comment:
I suppose this is true right now but it will change in the future?

Reply:
that's right! I was thinking we can keep the comment relevant, so we can update this comment as we add more to this. WDYT?